system/Log/Handlers/ChromeLoggerHandler.php

```diff
@@ -114,8 +114,10 @@ public function __construct(array $config = [])
 		$request = Services::request(null, true);
 
 		$this->json['request_uri'] = (string)$request->uri;
-
-		Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
+                
+                Events::on('post_controller', [$this, 'sendLogs'], EVENT_PRIORITY_HIGH);
+                
+		//Events::on('post_controller', [$this, 'sendLogs'], HOOKS_PRIORITY_HIGH);
```

Comment:
You're absolutely correct. It should be changed. And the constants themselves should be changed too. Thanks for catching that.